torchprime/torch_xla_models/configs/model/sharding/llama4-fsdp.yaml

```diff
@@ -1,17 +1,34 @@
+# Weights
+# vocab_size, hidden_size
 model.embed_tokens.weight: [fsdp, null]
+# hidden_size, num_attention_heads * head_dim
 model.layers.*.self_attn.q_proj.weight: [fsdp, null]
-model.layers.*.self_attn.k_proj.weight: [null, fsdp]
-model.layers.*.self_attn.v_proj.weight: [null, fsdp]
-model.layers.*.self_attn.o_proj.weight: [fsdp, null]
-model.layers.*.feed_forward.experts.gate_up_proj: [fsdp, null, null]
-model.layers.*.feed_forward.experts.down_proj:  [fsdp, null, null]
+# hidden_size, num_key_value_heads * head_dim
+model.layers.*.self_attn.k_proj.weight: [fsdp, null]
```

Comment:
any particular reason for changing this?

Reply:
good catch, this is a bug, mostly following this: https://github.com/AI-Hypercomputer/torchprime/blob/main/torchprime/torch_xla_models/configs/model/sharding/llama-fsdp.yaml